### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by molant**
This PR has quite a few commits. These are all more or less small issues that I've found while debugging www.tmz.com.

Most of the work to make it work will come in another PR related to the chrome debugging protocol

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "canvas-prebuilt": "^1.6.0-canvas.5",
     "chalk": "^2.0.0",
     "chrome-launcher": "^0.6.0",
-    "chrome-remote-interface": "^0.24.4",
+    "chrome-remote-interface": "^0.24.5",
```
Comment (alrra) [inline package.json]: Use `Upgrade:`  instead of `Chore:`, unless upgrading the dependency breaks something.

**Comment by alrra** [comment]
Merged all except https://github.com/sonarwhal/sonar/pull/486/commits/69c59599d8ef6471c727b765c6fb755eb34018b4, that creates some problems.

**Comment by molant** [comment]
> Merged all except 69c5959, that creates some problems.

I just saw that. Will take a look as soon as I can.

Thanks!

**Comment by molant** [comment]
@alrra, I fixed the problems with the redirect. Ready for another review.

**Pull request closed without merging**